### PR TITLE
DOC-7628 scope/collection analytics examples

### PR DESCRIPTION
**Pull request by osfameron**
The scope example is commented out, as I don't understand the
API here, have asked for advice from @avsej and will uncomment/fix in due course.

### CODE REVIEW

**Comment by osfameron** [comment]
Currently, uncommenting the handle-scope section gives this error:

```
.../.rvm/gems/ruby-3.0.0/gems/couchbase-3.0.3/lib/couchbase/scope.rb:102:in `analytics_query': uninitialized constant Couchbase::Scope::AnalyticsResult (NameError)
	from modules/howtos/examples/analytics.rb:91:in `<main>'
```
 This is with the following datasets created:
```
  CREATE DATAVERSE `travel-sample.inventory`;
  USE `travel-sample.inventory`;
  CREATE DATASET `airports-collection` ON `travel-sample`.inventory.airport;
```
This seems to match the conventions documented in the Java implementation of RFC, and does work in the Java SDK, so I'm not sure if the Ruby SDK is incomplete here?

File: modules/howtos/pages/analytics-using-sdk.adoc [hunk 1]
```diff
@@ -133,3 +133,27 @@ include::example$analytics.rb[tag=scanconsistency]
...
+[source,n1ql]
+----
+CREATE DATASET `airports-collection` ON `travel-sample`.inventory.airport;
+----
+
+You can run a query as follows:
+
+[source,ruby]
+----
+include::../examples/analytics.rb[tag=handle-collection]
```
Comment (RichardSmedley) [inline modules/howtos/pages/analytics-using-sdk.adoc:150]: I'm not sure what extra macro power Antora derives from using example$ in place of ../example/
(I'd need to rtfm ;) )
but I'm pretty sure that there was a reason beyond cargo-culting why we do this.

**Review by RichardSmedley — APPROVED** [review]
No problem merging from the adoc pov (although see passing comment on example$),
if @avsej has no time to review the code, I'm happy for it to be merged as working,
and we can revisit later if necessary.

**Comment by osfameron** [comment]
Thanks. I've made a few fixes including the `example$` one you mention, but in the correct branch (oops), so I'll close this PR and reissue, sorry for confusion.

**Comment by osfameron** [comment]
see #27

**Pull request closed without merging**